tests/scripts/thread-cert/node.py

```diff
@@ -1708,6 +1708,34 @@ def register_netdata(self):
         self.send_command('netdata register')
         self._expect_done()
 
+    def publish_netdata_dnssrp_anycast(self, seqnum):
```

Comment:
```suggestion
    def publish_netdata_dnssrp_anycast(self, seqnum):
```
```suggestion
    def netdata_publish_dnssrp_anycast(self, seqnum):
```
nit: Use comm prefix `netdata` ?

Reply:
Good suggestion. Will change it in next push.